docs/reference/data-streams/promote-data-stream-api.asciidoc

```diff
@@ -29,6 +29,14 @@ POST /_data_stream/_promote/my-data-stream
 
 `POST /_data_stream/_promote/<data-stream>`
 
+[[promote-data-stream-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `manage_ccr`
+<<privileges-list-cluster,cluster privilege>> to use this API. You must also
+have the `manage_follow_index` <<privileges-list-indices,index privilege>> for
+the local data stream.
```

Comment:
@martijnvg Can you help me confirm this? I had a hard time testing this locally.

Reply:
Thanks for pointing this out. So this api should require the same privileges as the unfollow api (which only requires `manage_follow_index` and not `manage_ccr`), however unfortunately the `manage_follow_index` privilege hasn't been updated to take the promote data stream api into account. I will fix this.

So current the `manage` privilege is required to use this api when security is enabled.

Reply:
I've opened #67773

Reply:
@jrodewig I've updated this docs snippet in #67773 to use `manage_follow_index` instead of `manage`.

Reply:
Awesome. Thank you @martijnvg!